apps/condo/domains/common/utils/serverSchema/resolveHelpers.js

```diff
@@ -0,0 +1,45 @@
+const { get } = require('lodash')
+
+const { FLAT_UNIT_TYPE, SECTION_SECTION_TYPE } = require('@condo/domains/property/constants/common')
+
+
+const getUnitTypeFieldResolveInput = ({ unitTypeFieldName = 'unitType', unitNameFieldName = 'unitName' } = {}) => ({ resolvedData, existingItem }) => {
+    const newItem = { ...existingItem, ...resolvedData }
+    const unitType = get(newItem, unitTypeFieldName)
+    const unitName = get(newItem, unitNameFieldName)
+    const existedUnitType = get(existingItem, unitTypeFieldName)
+
+    if (!unitName && unitType) {
+        return null
```

Comment:
Is it ok to reset `unitType` to null if no `unitName`?
What we should do if we need to send news items to all parkings?

Reply:
You can consider this to be default behavior.
You can redefine it to suit your needs.
NewsItem uses different logic for this field